src/ui/common/form/SwitchRenderer.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { Switch } from 'patternfly-react';
 
 const SwitchRenderer = ({ input }) => {
-  const switchValue = typeof input.value === 'string' ? (input.value === 'true') : input.value;
+  const switchValue = typeof input.value === 'string' ? (input.value === 'true' || input.value === 'active') : input.value;
```

Comment:
At the moment, SwitchRenderer only handles boolean values. If you want to extend the selection to other types, you can do something like:
```
const SwitchRenderer = ({ input, trueValue, falseValue }) => {
  const switchValue = input.value === 'true' || input.value === true || input.value === trueValue;
  return (<Switch
    {...input}
    value={switchValue}
    onChange={(el, val) => input.onChange(val ? trueValue : falseValue)}
  />);
};


SwitchRenderer.propTypes = {
  /* eslint-disable react/forbid-prop-types */
  trueValue: PropTypes.any,
  falseValue: PropTypes.any,
  /* eslint-enable react/forbid-prop-types */
  input: PropTypes.shape({
    value: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]).isRequired,
    onChange: PropTypes.func.isRequired,
  }).isRequired,
};

SwitchRenderer.defaultProps = {
  trueValue: true,
  falseValue: false,
};
```

and when using it:

```
<Field
  component={SwitchRenderer}
  trueValue="active"
  falseValue="inactive"
```